core/api/migrations/0002_initial.py

```
@@ -0,0 +1,36 @@
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="contributions",
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AddField(
+            model_name="contributions",
+            name="videos",
+            field=models.ManyToManyField(
+                related_name="contributions", to="api.contribution_videos"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)